hadoop-tools/hadoop-aws/pom.xml

```diff
@@ -479,6 +494,11 @@
       <artifactId>bundle</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>software.amazon.awssdk.crt</groupId>
+      <artifactId>aws-crt</artifactId>
+      <scope>compile</scope>
```

Comment:
provided, unless it really is to be shipped

Reply:
lets leave at provided, but docs to indicate this can be left out if the option is disabled

Reply:
do you mean leave at `compile` ? or should I move to provided? 

Also just to confirm, only difference is if it's provided it won't get included in the final release tar, and anyone who wants to use it has to put the jar there?